app/containers/MessageActions.js

```diff
@@ -293,6 +297,12 @@ export default class MessageActions extends React.Component {
 		toggleReactionPicker(actionMessage);
 	}
 
+	handleReport = async() => {
+		const { actionMessage } = this.props;
+		await RocketChat.reportMessage(actionMessage._id);
```

Comment:
wrap it in try-catch block and add showToast(error)

Reply:
sure

Reply:
@pranavpandey1998official Add an Alert instead of toast. (we're planning to remove it)
Also add try/catch.

Reply:
done